### ISSUE TITLE
"Empty language names are not allowed"

### ISSUE THREAD

**Issue by Codelica** [aziz/SublimeANSI#5]
Hello,

Using ST3 (build 3083) w/ ANSIescape installed via Package Manager.  When trying to use it as part of a build command I see the following error via popup and in console, and no ANSI coloring on the build output:

```
error: Error loading syntax file "Packages/User/ANSIescape/ANSI.tmTheme": Empty language names are not allowed
```

![ansi-error-st3](https://cloud.githubusercontent.com/assets/386101/9343466/66acfc9a-45bf-11e5-87dd-5717ef2f807d.png)

**Comment by matt1003** [comment]
@Codelica Are you still able to reproduce this issue?

**Comment by Codelica** [comment]
I'm on Sublime 3 build 3103 now and I just updated to the latest ANSIescape (v0.1.3).

It seems to work fine now viewing a file with escape codes after choosing "ANSI" for the file type.  

I can also see colored output for my Node.js build script that uses the "node" command.  

However for some reason my other Node.js build script using "nodemon" still shows the escaped characters rather then colors.  Perhaps that's because nodemon doesn't actually exit, it just sits waiting for further file changes.  Perhaps?

Thanks...

**Comment by jakkubu** [comment]
@Codelica Try to change ANSI_process_trigger to "on_data" in [`ansi.sublime-settings`](ansi.sublime-settings). This should solve the escaped characters during running deamon execution.

**Comment by matt1003** [comment]
@Codelica I have just released version 0.1.4 which adds the ANSIescape settings to the Sublime Text main menu. As jakkubu suggested, add the following to the user settings:

Preferences -> Package Settings -> ANSIescape -> Settings - User
`{`
`    "ANSI_process_trigger": "on_data"`
`}`

**Comment by Codelica** [comment]
Yep... working w/ Nodemon now. :+1:
